packages/manager/modules/vps/src/dashboard/components/vps-announcement-banner/controller.js

```diff
@@ -0,0 +1,12 @@
+import { TASKS_INFO } from './vps-announcement-banner.constant';
+
+export default class VpsAnnouncementBannerComponentController {
+  /* @ngInject */
+  constructor(coreConfig) {
+    this.coreConfig = coreConfig;
+  }
+
+  getTasksInfoLink() {
+    return TASKS_INFO[this.ovhSubsidiary] || TASKS_INFO.DEFAULT;
```

Comment:
👍

Reply:
Isn't supposed to be?

```suggestion
    return TASKS_INFO[this.coreConfig.getUser().ovhSubsidiary] || TASKS_INFO.DEFAULT;
```

Reply:
done